### ISSUE TITLE
Incorrect array stride in ByteBufferBasicBytez::setBoolean

### ISSUE THREAD

**Issue by abyrd** [RuedigerMoeller/fast-serialization#173]
The setBoolean method in ByteBufferBasicBytez currently reads:
```Java
@Override
    public void setBoolean(long byteIndex, boolean[] source, int elemoff, int siz) {
        for ( int i=0; i <siz; i++ ) {
            buffer.put((int) (byteIndex + 4 * i), (byte) (source[i + elemoff] ? 1 : 0));
        }
    }
```

This makes the width of a boolean in the output 4 bytes. The corresponding getBooleanArr method for reading boolean arrays uses a width of 1 byte, and all other setBoolean and getBooleanArr implementations in other Bytez subclasses use 1-byte widths.

**Comment by RuedigerMoeller** [comment]
thanks for your bug report

**Comment by RuedigerMoeller** [comment]
fixed with 2.49. Thanx for your help
